src/Utils/systems.jl

```diff
@@ -95,27 +106,42 @@ julia> dim(add_dimension(X))
 julia> typeof(X)
 LazySets.ZeroSet{Float64}
 ```
+
+More than one dimension can be added passing the second argument:
+
+```jldoctest add_dimension_set
+julia> Xext = add_dimension(BallInf(zeros(10), 0.1), 4);
+
+julia> dim(Xext)
+14
+```
+
+### Notes
+
+In the special case that the given set is a zero set, instead of cartesian product
+a new zero set with extended dimensions is returned.
 """
-function add_dimension(X::LazySet)::LazySet
+function add_dimension(X::LazySet, m=1)::LazySet
     if X isa ZeroSet
```

Comment:
I know this is old, but we should instead have two methods: one for `ZeroSet` and one for general `LazySet`.

On the other hand, we could even omit the `ZeroSet` case and rely on `LazySets` implementing `*` efficiently. But I guess we really need this here for handling empty inputs correctly, so let us keep it here.